ravenframework/SupervisedLearning/ARMA.py

```diff
@@ -461,10 +461,7 @@ def __setstate__(self, d):
     try:
       self.randomEng
     except AttributeError:  # catches where ARMA was pickled without saving the RNG
-      self.setEngine(randomUtils.newRNG(), seed=None, count=rngCounts)
-    else:
-      if isinstance(self.randomEng, findCrowModule('randomENG').RandomClass):
-        self.randomEng = randomUtils.RNG(self.randomEng, self.seed)  # wraps raw crow RNG
+      self.setEngine(engine=randomUtils.newRNG(), seed=self.seed, count=rngCounts)
```

Comment:
"engine" is not a valid keyword here, either do not use it or use "eng" instead, since you are using "eng" in setEngine method.

Reply:
Fixed. Sorry, I feel like this has become the never ending code review for you @wangcj05.